src/rest_framework_dso/filters.py

```diff
@@ -571,10 +571,12 @@ def to_html(self, request, queryset, view):
         """
         return ""
 
+    _NON_FIELDS = {"_fields", "_format", "_sort", "_pageSize", "_page_size"}
+
     def is_unfiltered(self, request):
         # If there are request parameters (except for this hard-coded exclude list),
         # they are assumed to be filters.
-        return {"_format", "_sort", "_pageSize", "_page_size"}.issuperset(request.GET.keys())
+        return request.GET.keys() <= self._NON_FIELDS
```

Comment:
ok, cool. hoe werkt dit?

Reply:
`set.__le__` is hetzelfde als `set.issubset`:

```python
>>> {1,2,3} <= {1,2,3}
True
>>> {1,2,3} <= {1,2,3,4}
True
>>> {1,2,3} <= {1,2}
False
```

Het scheelt één paar haakjes, en omdat de verzameling groter werd, moest ik de regel toch al opbreken.